### ISSUE TITLE
Contingency Tables

### ISSUE THREAD

**Issue by nataliekatsaros** [nataliekatsaros/QMB210-15-Group3#5]

**Comment by Luccaloffreno** [comment]
Looking at the tables a lot of the juniors and seniors decide they don't want the meal plan and rather do something else

**Comment by nataliekatsaros** [comment]
> Looking at the tables a lot of the juniors and seniors decide they don't want the meal plan and rather do something else

Yes, I noticed that too. Expand on that in your project and see how UT can retain students to keep the meal plan.

**Comment by nataliekatsaros** [comment]
> Looking at the tables a lot of the juniors and seniors decide they don't want the meal plan and rather do something else

How's your video recording going?
